essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/e2etest/Sep6End2EndTest.kt

```diff
@@ -138,7 +146,13 @@ open class Sep6End2EndTest : AbstractIntegrationTests(TestConfig()) {
 
     // Supply missing financial account info to continue with the transaction
     val additionalRequiredFields =
-      sep6Client.getTransaction(mapOf("id" to withdraw.id)).transaction.requiredCustomerInfoUpdates
+      anchor
+        .customer(token)
+        .get(id = customer.id, memo = memo, transactionId = withdraw.id)
```

Comment:
Is it required to provide all 3? From my understanding transactionId should be enough

Reply:
It shouldn't require all 3. Thanks for catching it.